health/module_test.go

```diff
@@ -0,0 +1,33 @@
+package health
+
+import (
+	"context"
+	"testing"
+
+	"github.com/textileio/powergate/net/lotus"
+	"github.com/textileio/powergate/tests"
+	"github.com/textileio/powergate/tests/mocks"
+)
+
+var (
+	ctx = context.Background()
+)
+
+func TestModule(t *testing.T) {
+	t.Parallel()
```

Comment:
Nit: I'm usually avoiding the parallel flag on tests that require local devnets since puts a lot of stress to the CI runner.